### ISSUE TITLE
[Feature Request] Monthly/weekly incremental chart

### ISSUE THREAD

**Issue by tonyqus** [dotnet/nuget-trends#182]
Hi Team,

Thank you for this nice tool.  I'm using the chart to evaluate the growth of community for different .NET packages. However, I always see something is increasing because the download hits will definitely increase all the time.

How about we create a monthly/weekly incremental chart? This can help developers understand what's the difference between each month.

**Comment by bruno-garcia** [comment]
Totally agreed it would be very beneficial to see the change. Is this something you're willing to contribute with a PR? We'll be happy to review/merge/release the change

**Comment by tonyqus** [comment]
Frankly speaking, I don't have much time on contributing this project. Is your team short of hand? As an open source project maintainer, I totally understand how hard to get a new hand from community to contribute. Open source project is just like a lonely cowboy story.

**Comment by joaopgrassi** [comment]
Yeah it's tough. It has been pretty much Bruno and I maintaining it. I can't say for Bruno, but I don't really have the time now, so if it's something you can do it, we would appreciate :).

**Comment by FatTigerWang** [comment]
@joaopgrassi  I have free time to do this, but I don't know where to change it, any guidance would be greatly appreciated

**Comment by FatTigerWang** [comment]
Can be assigned to me if possible.
